iidm-actions-contingencies-xml-client/src/test/java/eu/itesla_project/iidm/actions_contingencies/xml/test/PstOperationTest.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.iidm.actions_contingencies.xml.test;
+
+import eu.itesla_project.iidm.actions_contingencies.xml.XmlFileContingenciesAndActionsDatabaseClient;
+import eu.itesla_project.iidm.network.Network;
+import eu.itesla_project.iidm.network.test.PhaseShifterTestCaseFactory;
+import eu.itesla_project.modules.contingencies.Action;
+import eu.itesla_project.modules.contingencies.ActionElement;
+import eu.itesla_project.modules.contingencies.ActionElementType;
+import eu.itesla_project.modules.contingencies.TapChangeAction;
+
+import org.junit.Test;
+import org.xml.sax.SAXException;
+
+import javax.xml.bind.JAXBException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author Mathieu Bague <mathieu.bague at rte-france.com>
+ */
+public class PstOperationTest {
+
+    @Test
+    public void test() throws JAXBException, SAXException {
+        Network network = PhaseShifterTestCaseFactory.create();
+
+        Path path = Paths.get("src/test/resources/pstOperations.xml");
```

Comment:
Use getClass().getResourceAsStream("pstOperation.xml") instead to not depend on the source layout.